domain-model

```diff
@@ -0,0 +1,64 @@
+As an air traffic controller
+So I can get passengers to a destination
+I want to instruct the airport to land a plane
+Objects||		 	||Properties			||Messages					||Output
+
+Airport				|| planes @array   		||getNumberOfPlane()		||@boolean
+Plane				||planeToAdd @number	||landPlane()	
+
+
+
+			
+As the system designer
+So that the software can be used for many different airports
+I would like a default airport capacity that can be overridden as appropriate
+
+Objects||		 	||Properties			||Messages					||Output
+
+Airport				|| planes @array   		||increaseCapacity()		||@boolean @voidPlane				||planeToAdd @number		
+
+
+			
+As an air traffic controller
+To ensure safety
+I want to prevent landing when the airport is full
+Objects		 ||Properties					 ||Messages	      				||Output
+Airport															
+Plane	 	 ||airport [@array] @number 	 ||isAirportFulll()				||@boolean @void
+
+
+			
+As an air traffic controller
+So I can get passengers on the way to their destination
+I want to instruct the airport to let a plane take off and confirm that it is no longer in the airport
+
+Objects	||		||Properties			||Messages						||Output
+
+Airport	||		||planes@array @number	||instructPlaneTakeOFf()
+plane	||								||confirmTakeOff()				||@boolean @void
+
+
+			
+As an air traffic controller
+To avoid confusion
+I want to prevent asking the airport to let planes take-off which are not at the airport, or land a plane that's already landed
+
+Objects	||		||Properties			||Messages						||Output
+
+Airport	||		||planes@array @number	||confirmLand()
+plane	||								||PreventPlaneTakeOff()			||@boolean @void
+
+
+			
+Extended Acceptance Criteria
+As an air traffic controller
+To ensure safety
+I want to prevent takeoff when weather is stormy
+
+As an air traffic controller
+To ensure safety
+I want to prevent landing when weather is stormy
+
+As an air traffic controller
+To count planes easily
+Planes that have landed must be at an airport
```

Comment:
i do not like that my domain model looks bad. Hence, i will improve on writing better formatting and ensuring it is easy to read.